README.md

```diff
@@ -0,0 +1,32 @@
+# squashdelta
+Create efficient deltas (patches) between two SquashFS images
+
+## Building from source
+This project uses [autotools](http://inti.sourceforge.net/tutorial/libinti/autotoolsproject.html)
```

Comment:
I'm against linking to some project's documentation that is in no way related to this project or upstream autotools.